airbyte-webapp/scripts/regenerate-api-clients.ts

```diff
@@ -0,0 +1,13 @@
+import { statSync, existsSync } from "fs";
+import { resolve } from "path";
+
+import conf from "../orval.config";
+
+const lastTouched = (filepath: string) => statSync(resolve(filepath)).mtime;
```

Comment:
Did you test what happens if the file doesn't exist?

Reply:
It breaks, as you presumably guessed. I added the `existsSync` check below, on line 9, after seeing the build step fail in CI without it (I thought that would happen, and it did! Hooray for validating guesses.)